src/devtools/devtools.tsx

```diff
@@ -85,7 +85,7 @@ const theme = {
   success: '#00ab52',
   danger: '#ff0085',
   active: '#006bff',
-  warning: '#ffb200',
+  warning: '#ff6200',
```

Comment:
This works! However, I think we could probably just make the text black for that button like the rest of the tools.

Reply:
For All other buttons? or just for the new one,
because it is kinda off having this button with different color (inconsistent)

Reply:
Anything with the yellow background should have black text I think.

Reply:
Got it 👍🏻, What do you thing if it was moved to this place to make it more consistent 👇 
![image](https://user-images.githubusercontent.com/43112535/102931272-23788a80-44af-11eb-97b1-fd053fbdbe25.png)

Reply:
The order before was from most useful to least. What is this order inspired from?

Reply:
White text buttons then the black one 
if it makes bad DX it shall remain as is

Reply:
Sorry for making this PR as long as this, i wanted to make a good one 😅

Reply:
I personally like the order as is. And no worries on the PR! These are the interactions that bring us all closer :)